name.abuchen.portfolio.tests/src/name/abuchen/portfolio/snapshot/security/DividendCalculationTest.java

```diff
@@ -12,8 +13,10 @@
 import name.abuchen.portfolio.junit.TestCurrencyConverter;
 import name.abuchen.portfolio.model.Account;
 import name.abuchen.portfolio.model.AccountTransaction;
+import name.abuchen.portfolio.model.AccountTransaction.AccountTransactionBuilder;
 import name.abuchen.portfolio.model.Portfolio;
 import name.abuchen.portfolio.model.PortfolioTransaction;
+import name.abuchen.portfolio.model.PortfolioTransaction.PortfolioTransactionBuilder;
```

Comment:
Are you duplicating functionality from `name.abuchen.portfolio.junit/src/name/abuchen/portfolio/junit/AccountBuilder.java`?

Reply:
Oh yes a little bit. AccountBuilder is technical debt. The Builder should be have the same name then the original class and it needs to be a part of the class that the builder construct. Then all people can find and use it. I will mark the AccoutBuilder as deprecated.

Reply:
done

Reply:
It doesn’t work like that. You may not like the current `AccountBuilder` and `PortfolioBuilder`, and others may not like it for different reasons, but not everyone can add his/her own, additional version. If you think you can improve them, do so, but then you need to update all current uses as well.